style.css

```diff
@@ -0,0 +1,271 @@
+.logo {
+  width: 76px;
+  height: 20px;
+  font-family: 'poppins', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #6070ff;
+  flex: none;
+  order: none;
+  flex-grow: 1;
+  border: rgb(247, 244, 244);
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+#bttn {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #396df2;
+  padding: 12px;
+  background: #fff;
+  border: 1px solid #396df2;
+  border-radius: 12px;
+}
+
+#bttn:active {
+  background: #2e3de9;
+  color: #fff;
+}
+
+#bttn:hover {
+  background: #525ec4;
+  color: #fff;
+}
+
+.list {
+  list-style: none;
+  display: flex;
+  justify-content: stretch;
+  align-items: flex-end;
+}
+
+.canopy {
+  list-style: none;
+}
+
+a { text-decoration: none; }
+
+.text {
+  margin: 0 24px;
+  width: 327px;
+  height: 150px;
+  font-family: 'poppins', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  align-items: center;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  display: flex;
+  justify-content: space-evenly;
+  list-style: none;
+}
+
+.head-section {
+  width: 327px;
+  height: 146px;
+  font-family: 'poppins', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.main-section {
+  align-items: center;
+  padding-left: 10px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  padding-top: 10%;
+  padding-bottom: 10%;
+  line-height: 52px;
+  flex-direction: column;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+}
+
+.container {
+  display: grid;
+  grid-template-columns: 100%;
+  grid-template-rows: repeat(5, auto);
+}
+
+.card_item {
+  border: 1px solid rgb(241, 234, 234);
+  border-radius: 10px;
+  padding: 10px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
```

Comment:
- Kindly, your card item is not styled like the Figma one. the button, teck lists, font, the border has to be styled, 👌👍


| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/52010789/206655095-4cffbbdc-b7b0-45ed-81a1-c5b9ed4c6420.png" width="200">|<img src="https://user-images.githubusercontent.com/52010789/206655105-9879a39a-9d9a-45a6-8f1e-2ea87f134953.png" width="200">|


😎️